charts/spinnaker/templates/configmap/halyard-config.yaml

```diff
@@ -111,8 +111,14 @@ data:
     $HAL_COMMAND config provider kubernetes enable
     {{- range $name, $account := .Values.kubeConfig.accounts }}
 
+    # Create a script to upsert each account. This enables parallel execution later on.
+    # N.B.: \$ is used to not apply variable substitution, as those commands must be executed in
+    #       context of the script.
     cat > /tmp/upsert_{{ regexReplaceAll "\\.|-" $name "_" }} <<- EOF
-    if $HAL_COMMAND config provider kubernetes account get {{ $name }}; then
+
+    ACCOUNT="\$($HAL_COMMAND config provider kubernetes account get {{ $name }} --no-validate --quiet --output=json)"
+    # The account exists already.
+    if [ "\${?}" -eq 0 ]; then
```

Comment:
I guess maybe we can avoid this by checking if `ACCOUNT` is the empty string or not. 🤔